tests/infra/logging_app.py

```diff
@@ -138,6 +139,7 @@ def issue(
                     if self.scope is not None:
                         args["scope"] = self.scope
                     url = "/app/log/private"
+                    url = private_url if private_url else url
```

Comment:
```suggestion
                    url = url or "/app/log/private"
```

Reply:
I believe this suggestion is applicable only if I change the param name to `url` from `private_url`? (Other than replacing the if else semantic)

Reply:
Yes, this is just a "Python can do it in one line" comment :)

Reply:
Gotcha, will remember this for next time :)